routed-echo/README.md

```diff
@@ -0,0 +1,52 @@
+# Routed Host: echo client/server
+
+This example is intended to follow up the basic host and echo examples by adding use of the ipfs distributed hash table to lookup peers.
+
+Functionally, this example works similarly to the echo example, however setup of the host includes wrapping it with a Kademila hash table, so it can find peers using only their IDs. 
+
+We'll also enable NAT port mapping to illustrate the setup, although it isn't guaranteed to actually be used to make the connections.  Additionally, this example uses the newer `bhost.NewHost` constructor.
```

Comment:
I don't see bhost.NewHost anywhere :)

Reply:
hmm, your right.  I'll sort it out, sorry